devices/lidl.js

```diff
@@ -216,4 +217,16 @@ module.exports = [
             device.getEndpoint(1).saveClusterAttributeKeyValue('lightingColorCtrl', {colorCapabilities: 29});
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_htnnfasr'}],
+        model: 'PSBZS A1',
+        vendor: 'Parkside',
```

Comment:
Please change this to Lidl and change the description to `'Parkside smart watering timer'`

Reply:
done